.gitignore

```diff
@@ -20,3 +20,6 @@ Build
 *.swp
 *.orig
 .arcconfig
+
+*.tgz
+build.ninjae
```

Comment:
Instead of doing this, why not change the sed script so that you're using an already known extension like ~ which will be ignored? Then you don't need to add this.